static/templates/compose.hbs

```diff
@@ -90,16 +90,20 @@
                             </div>
                             <div class="drag"></div>
                             <div id="below-compose-content">
-                                <button type="submit" id="compose-send-button" class="button small send_message animated-purple-button" title="{{t 'Send' }} (Ctrl + Enter)">{{t 'Send' }}</button>
+                                <div class="compose_right_float_container">
+                                    <button type="submit" id="compose-send-button" class="button small send_message animated-purple-button hide-mobile" title="{{t 'Send' }} (Ctrl + Enter)">{{t 'Send' }}</button>
+                                </div>
                                 {{> compose_control_buttons }}
-                                <span id="sending-indicator"></span>
-                                <div id="send_controls" class="new-style">
-                                    <label id="enter-sends-label" class="compose_checkbox_label checkbox">
-                                        <input type="checkbox" id="enter_sends" />
-                                        <span></span>{{t 'Press Enter to send' }}
-                                    </label>
+                                <div class="compose_right_float_container">
+                                    <span id="compose_limit_indicator"></span>
+                                    <div id="send_controls" class="new-style">
+                                        <label id="enter-sends-label" class="compose_checkbox_label checkbox">
+                                            <input type="checkbox" tabindex="0" id="enter_sends" />
+                                            <span></span>{{t 'Press Enter to send' }}
+                                        </label>
+                                    </div>
+                                    <button type="submit" id="compose-send-button" class="button small send_message animated-purple-button show-mobile" title="{{t 'Send' }} (Ctrl + Enter)">{{t 'Send' }}</button>
```

Comment:
We have two send buttons now, one is displayed on mobile widths and other on wide screens. I don't think we need to worry about `tab` on mobile widths?

This is purely for alignment of send button naturally.

Reply:
Oh! This is not good. I will actually just align it properly on mobile using css.